datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -119,7 +119,7 @@ def resolve_db_host(db_host):
 
     try:
         host_ip = socket.gethostbyname(db_host)
-    except socket.gaierror as e:
+    except (socket.gaierror, UnicodeError) as e:
```

Comment:
Strange that it's a `UnicodeError` for "hostname too long". Looks like this is a long-standing issue: https://bugs.python.org/issue32958